DevEdu.API/DevEdu.Business.Tests/Group/GroupServiceTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿using DevEdu.Business.Services;
+using DevEdu.DAL.Models;
+using DevEdu.DAL.Repositories;
+using Moq;
+using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace DevEdu.Business.Tests.Group
+{
+    public class GroupServiceTests
+    {
+
+        private Mock<IGroupRepository> _mock;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mock = new Mock<IGroupRepository>();
+        }
+
+        [TestCaseSource(typeof(GroupServiceExpecteds), nameof(GroupServiceExpecteds.AddGroup))]
+        public void AddGroup(GroupDto dto, GroupDto expected)
+        {
+            //Given
+            _mock.Setup(mock => mock.AddGroup(dto)).Returns(expected);
+            var _service = new GroupService(_mock.Object);
+            
+            //When
+            var actual = _service.AddGroup(dto);
```

Comment:
please provide an option to create a group with list of teachers and list of tutors (because it's a real business-case)